### ISSUE TITLE
generated headers installed to wrong location

### ISSUE THREAD

**Issue by rakslice** [mej/libast#2]
`libast`'s generated headers `sysdefs.h` and `types.h` are installed directly in `$PREFIX/include` rather than in `$PREFIX/include/libast`, and I believe the latter was intended as that's where the other headers are installed.  

`include/Makefile.am` specifies the generated headers as:
```
nodist_include_HEADERS = libast/sysdefs.h libast/types.
```
(i.e. not `nobase_*` which would direct to preserve the directory name)

So `make install` does e.g.:
```
 /usr/bin/install -c -m 644 libast/sysdefs.h libast/types.h '/usr/local/include'
```
which installs the files to e.g. `/usr/local/include/sysdefs.h` and `/usr/local/include/types.h`.

This not only results in headers that are not findable, but also could clobber some pretty important existing system headers depending on the value of `$PREFIX`.

**Comment by rakslice** [comment]
Relevant docs at https://www.gnu.org/software/automake/manual/automake.html#index-nobase_005f-and-dist_005f-or-nodist_005f

**Comment by rakslice** [comment]
Looks like this was fixed in c6a008f51b38014e7620937bd13ca312dfdeeed5
